src/Alert.jl

```diff
@@ -0,0 +1,268 @@
+###################################################
+#
+# Copyright © Akamai Technologies. All rights reserved.
+# Proprietary and confidential.
```

Comment:
take this out.  it's opensource, so it's neither proprietary nor confidential

Reply:
deleted `# Proprietary and confidential.`.  Should we delete `Copyright...` as well?

Reply:
No we most definitely want to keep the copyright.  All opensource code has a copyright otherwise you can't license it. 

This is what it should look like:
```
###################################################
#
# Copyright 2016 SOASTA, Inc.
# Distributed under the terms of the MIT license
#
# File: mPulseAPI.jl
#
# Functions to communicate with the mPulse Query and Repository REST APIs
#
###################################################
```